src/JuliusSweetland.OptiKey.Core/UI/ViewModels/MainViewModel.ServiceEventHandlers.cs

```diff
@@ -1128,10 +1128,6 @@ private void HandleFunctionKeySelectionResult(KeyValue singleKeyValue)
                     SelectLanguage(Languages.HebrewIsrael);
                     break;
 
-                case FunctionKeys.HungarianHungary:

```

Comment:
Oops. Don't worry, I put it back in.

Reply:
Could you pls update the PR with this reinstated? It is quite hard to keep track of what is/is not in a logical change if it spans multiple PRs. Thanks.

Reply:
Done.